src/Views/AppInfoView.vala

```diff
@@ -583,10 +598,9 @@ namespace AppCenter.Views {
             }
         }
 
-        class UrlButton : Gtk.Button {
-            public UrlButton (string label, string uri, string icon_name) {
+        class UrlButton : Gtk.Grid {
```

Comment:
This changes the `UrlButton` to a grid with an optional URI so that we can reuse it whether or not a URL is set. It also feels like an abuse, and I forget off the top of my head why it was done.